src/index.js

```diff
@@ -1,6 +1,7 @@
 import $ from 'jquery';
 import {
-  setMobileMenuFullscreen, toggleMobileMenu, hideMobileMenuForDesktop, hideMobileMenuOnClick
+  setMobileMenuFullscreen, toggleMobileMenu, hideMobileMenuForDesktop, hideMobileMenuOnClick,
+  setupBanner
```

Comment:
I do not find very readable. Should we add `"object-curly-newline": ["error", {"ImportDeclaration": "never"}],` to `.eslintrc`?

- [ ] object-curly-newline